src/fctGraph/Utils/mutators/addOutputFctForOutSlotWhichHasNone.js

```diff
@@ -41,6 +41,10 @@ function safeAddOutputFctAndConnect(fctGraph, outSlot, outputFct) {
 function addOutputFctForOutSlotWhichHasNone(fctGraph, outSlot, subTypeClass, opts) {
   assertPrerequisites(fctGraph, outSlot)
 
+  if ("customData" in opts) {
+    console.warn('DEPRECATION WARNING: "customData" key is deprecated. Please use "fctData" instead')
+  }
+
```

Comment:
I would even consider throwing here instead of letting execution continue. if one used `customData`, they will end up with a result that doesn't look like they expected.

Reply:
But that would also break our hambda implementation. 

`customData` usage hasn't changed yet. If they pass it, it will be applied.